websrc/source/documents/tutorials/prometheus-tutorial.md

```diff
@@ -0,0 +1,145 @@
+---
+layout: "documents"
+page_title: "Contiv Prometheus Integration for Kubernetes"
+sidebar_current: "prometheus_tutorial"
+description: |-
+  Contiv Prometheus Integration
+---
+
+
+
+
+## Data Analytics with Prometheus and Grafana for Kubernetes
+
+- [Setup](#setup)
+- [Chapter 1 - Prometheus and Grafana](#ch1)
+- [Chapter 2 - Basic Ping Example](#ch2)
+- [Resources](#resources)
+
+This walks through the setup for Prometheus and Grafana and an example that may be useful for troubleshooting connectivity issues within your cluster.
+
+### <a name="setup"></a> Setup
+
+Please finish the [Container Networking Tutorial](https://contiv.github.io/documents/tutorials/networking-kubernetes-16.html), but do not clean up your cluster.
+
+In order to install Prometheus and Grafana, please follow these steps.
+
+```
+[vagrant@kubeadm-master ~]$ cd contiv-*
+[vagrant@kubeadm-master contiv-1.1.1]$  cd install/k8s/k8s1.6
+[vagrant@kubeadm-master k8s1.6]$ sudo bash
```

Comment:
Use `sudo -i` to become root

The end result is (usually) the same, but `-i` simulates a login shell and results in execution of scripts which other methods of becoming root do not